### DISCUSSION TITLE
IGV Automation Commands

### DISCUSSION THREAD

**Discussion by OfficerLahey** [biostars/biostar-handbook-issues#257] [Q&A] [answered]
I was going through the "Using the Integrative Genomics Viewers" section of the Biostar handbook and I got to the section on automating IGV. I created the FASTA and gff file and using the app and loading from inside the app works but it will not work from the command line. It loads the sequence but nothing else.

![image](https://user-images.githubusercontent.com/45005333/226056693-c414cc24-1929-4d32-bfff-6bb4911605ff.png)

* Using Windows Bash
* Works when loading directly in the app.
* The bam file does not exist because I did not create it but the .gff is there and should load.
* I refreshed the file explorer as well and it still says it does not exist.
* I also copied and pasted the entire file path extension and it still did not work

Any ideas would be appreciated. I am frustrated by this.

Thanks.

**Comment by ialbert** [comment, marked as answer]
I suspect that the problem is that IGV runs in Windows, and the commands run from the Linux subsystem

the solution would be to pass the same path to the file as it is from Windows. 

In Explorer, navigate to the file in question hold Shift, right-click, and select "Copy as path". That would be the name that you need to put as path to the file.

https://www.howtogeek.com/670447/how-to-copy-the-full-path-of-a-file-on-windows-10/

**Reply by OfficerLahey** [reply]
That fixed it! Thank you!
